### ISSUE TITLE
Why does my crf.accuracy increase so fast?

### ISSUE THREAD

**Issue by buptcjj** [keras-team/keras-contrib#321]
I use BiLSTM+CRF to do a sequence tagging model for postagging. But I find my crf.accuracy increases so fast and reach 1.0 just in the first epoch

```python
        output = TimeDistributed(Dense(100,activation='relu'),name='Dense_1')(lstm_output)
        output = Dropout(rate=0.3)(output)
        if self.use_crf:
            crf_layer = CRF(self.label_num,sparse_target=True)
            output = crf_layer(output)
            model = Model(inputs, output)
            model.compile('adam', loss=crf_layer.loss_function, metrics=[crf_layer.accuracy])
        else:
            output = TimeDistributed(Dense(self.label_num,activation='softmax'), name='Dense_2')(output)
            model = Model(inputs, output)
            model.compile('adam', loss='categorical_crossentropy', metrics=['accuracy'])``` 

the crf_layer.accuracy seems to increase so fast. In the first epoch, it shows as
 1888/2000 [===========================>..] - ETA: 4s - loss: 17.2046 - acc: 0.9666
 1920/2000 [===========================>..] - ETA: 3s - loss: 17.1906 - acc: 0.9671
 1952/2000 [============================>.] - ETA: 2s - loss: 17.2806 - acc: 0.9677
 1984/2000 [============================>.] - ETA: 0s - loss: 17.2222 - acc: 0.9682

But the accuracyis is just 0.4 if I don't use CRF, just like the code in else part, and the training looks more normal. 
So whether I ignore some details？
